src/lib/mina_base/coinbase.ml

```diff
@@ -86,16 +86,28 @@ module Gen = struct
     let%bind supercharged_coinbase = Quickcheck.Generator.bool in
     let%bind amount =
       let max_amount = constraint_constants.coinbase_amount in
-      let%map amount = Currency.Amount.(gen_incl zero max_amount) in
+      let min_fee =
+        Option.value_exn
+          (Currency.Fee.scale constraint_constants.account_creation_fee 2)
```

Comment:
Can you add a comment here, so that it's obvious that this is due to the `max_fee` below?